ui/src/components/textInputDialog.tsx

```diff
@@ -8,6 +8,7 @@ import { ChangeEvent, ReactNode, useState } from "react";
 
 type textInputDialogProps = {
   title: string;
+  text?: string;
```

Comment:
How about `initialText` to make it clear that it only applies when the dialog first appears?

Reply:
Yes. It think you are right. Only the initial text